### DISCUSSION TITLE
Julia Markdown from Weave.jl

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [quarto-dev/quarto-cli#397] [Q&A] [answered]
Hi all,

Thanks for the hard work on Quarto! I just came across [Weave.jl](https://github.com/JunoLab/Weave.jl) and thought It could be useful and something to leverage from to improve vscode-extension for Julia.

Cheers.

**Comment by jjallaire** [comment, marked as answer]
Hi there, thanks! We currently integrate w/ [IJulia](https://github.com/JuliaLang/IJulia.jl) (i.e. if you have a Jupyter notebook using the Julia kernel or a qmd file with a `{julia}` cell that should work fine). 

We also recently enhanced our VS Code plugin (https://marketplace.visualstudio.com/items?itemName=quarto.quarto) w/ support for executing Julia cells + code completion within Julia cells. We'll be adding more documentation on the Quarto website soon on how to use Julia w/ Quarto.

**Reply by camilogarciabotero** [reply]
Hi @jjallaire 

Thanks for your reply. I've been playing around with the extension and it is pretty advanced now! I just experienced an [issue](https://github.com/quarto-dev/quarto-julia/issues/1#issue-1161732537) when knitting a `qmd` file with a Julia chunk, yet it is pretty versatile and responsive!.

Thanks.

**Comment by jjallaire** [comment]
FYI we've added some additional documentation on using Julia w/ Quarto here: https://quarto.org/docs/computations/julia.html

**Reply by camilogarciabotero** [reply]
Thank you!
